ambry-api/src/main/java/com/github/ambry/router/ReadableStreamChannel.java

```diff
@@ -31,7 +31,7 @@ public interface ReadableStreamChannel extends Channel {
    * Return the size of stream that is available on this channel. If -1, then size is unknown.
    * @return the size of the stream available on this channel. -1 if size is unknown.
    */
-  public long getSize();
+  long getSize();
```

Comment:
why remove public ?

Reply:
It's an interface.  Everything is public by default.  The "public" has no impact.